samples/ipc/ipc_service/README.rst

```diff
@@ -82,17 +89,16 @@ Use these overlays when building the IPC sample to test the following scenarios:
 
 **nRF54H20 DK**
 
-You can build the sample to test IPC between the application and PPR core using the ICMSG backend, as follows:
+You can build the sample to test IPC between the application and PPR core using the ICBMSG backend, as follows:
```

Comment:
```suggestion
You can build the sample to test IPC between the application and PPR core using the ``icbmsg`` backend, as follows:
```
In other places we use lower case latters.
Or consider adding a reference to `ipc_service_backend_icbmsg`.
@peknis what do you think about it?

Reply:
I'd prefer having the link.

Reply:
I agree. Changed to link